src/reducetensor.cpp

```diff
@@ -568,7 +569,7 @@ std::size_t ReduceTensorDescriptor::GetWorkspaceSize(const Handle& handle,
     std::size_t wsSizeInBytes =
         !need_indices ? workspace_size * detail::GetDataTypeSize(inDesc.GetType())
                       : workspace_size * (detail::GetDataTypeSize(inDesc.GetType()) + sizeof(int)) +
-                            64 + sizeof(int);
+                            64 + sizeof(int) + 64;
```

Comment:
define just above `ReduceTensorDescriptor::GetWorkspaceSize()`:
```C++
constexpr std::size_t workspaceAlignRequirementBytes = 4; // or 64 if you wish so but I do not see any reason
```
then
```suggestion
                            64 + sizeof(int) + workspaceAlignRequirementBytes;
```